### PR TITLE
[Y_Demkiv] CSS Practice Task 1

### PR DESCRIPTION
**Pull request by YarynaD**
https://yarynad.github.io/ui-internship-lwo2/css-practice/task%201/index.html

### CODE REVIEW

**Comment by tarasbobak** [comment]
HTML is invalid
https://validator.w3.org/#validate_by_input

File: css-practice/task 1/index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <header>
+        <h1 class="Logo">MINIMØ</h1>
+        <nav class="Navigation_panel">
+            <a href="page.com" class="Nav_item">lifestyle</a>
+            <a href="page.com" class="Nav_item">photodiary</a>
+            <a href="page.com" class="Nav_item">music</a>
+            <a href="page.com" class="Nav_item">travel</a>
+        </nav>
+    </header>
+    <section class="Banner"></section>
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: if it's only banner it doesn't have any semantic meaning. It's not a section.
http://html5doctor.com/the-section-element/

File: css-practice/task 1/index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            <a href="#"><h6>Leave a comment</h6></a>
+        </article>
+        <section class="Newsfeed">
+            <article class="News">
+                <img src="img/image1.png" alt="">
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: image should have an alt text

File: css-practice/task 1/index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            <a href="page.com" class="Nav_item">photodiary</a>
+            <a href="page.com" class="Nav_item">music</a>
+            <a href="page.com" class="Nav_item">travel</a>
+        </nav>
+    </header>
+    <section class="Banner"></section>
+    <main>
+        <article class="Intro">
+            <h6>photodiary</h6>
+            <a href="#"><h2>The perfect weekend getaway</h2></a>
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: No block elements inside inline!

File: css-practice/task 1/index.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        </section>
+    </main>
+    <aside class="Feedback">
+        <h2>Sign up for our newsletter!</h2>
+        <form action="#">
+            <input type="text" placeholder="Enter a valid email address">
+            <button></button>
+        </form>
+    </aside>
+    <aside class="OtherNews">
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: className can look like hello-world not hello_world, not HelloWorld, not helloWorld

File: css-practice/task 1/index.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        <section class="RandomNews">
+            <div class="News">
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: Why on top you used article, but here div?

File: css-practice/task 1/index.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+            </div>
+        </section>
+        <button>Load more</button>
+    </aside>
+    <footer>
+        <div id="Terms">
+            <a href="">Terms and conditions</a>
+            <a href="">Privacy</a>
+        </div>
+        <div id="media">
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: why do you use id's. Try yo use only classes

File: css-practice/task 1/index.html [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+            <a href="">Follow</a>
+            <a id="fb" href=""><img src="img/facebook.png" alt="fb"></a>
+            <a id="twiter" href="#"><img src="img/twitter.png" alt="twit"></a>
+            <a id="instagram" href="#"><img src="img/instagram.png" alt="instagram" sizes="" srcset=""></a>
```
Comment (tarasbobak) [inline css-practice/task 1/index.html]: why do you need sizes and srcset???

File: css-practice/task 1/style.css [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+    margin-left: 70px;
+    margin-right: 70px;
+}
+
+footer {
+    width: 100%;
+    height: 57px;
+    background-color: #f0f0f0;
+    padding-top: 25px;
+    margin-bottom: 1px;
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: 1px?)))

File: css-practice/task 1/style.css [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+    font-family: "Playfair Display";
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #626262;
+}
+
+.Banner {
+    background-color: black;
+    background: url(img/image.png);
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: this is wrong usage of rules. Of you use specific background rules then you should have used background-image and not generic background here.

File: css-practice/task 1/style.css [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: try to use generic background rules for all your specific rules.

File: css-practice/task 1/style.css [hunk 11]
```diff
@@ -0,0 +1,223 @@
...
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 6px;
+}
+
+.Nav_item {
+    color: #626262;
+    font-family: 'Playfair Display', serif;
+    font-size: 16px;
+    font-weight: 400;
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: isn't it a default value?

File: css-practice/task 1/style.css [hunk 12]
```diff
@@ -0,0 +1,223 @@
...
+    width: 100%;
+    height: 200px;
+    background-color: #f0f0f0;
+    padding-top: 35px;
+    padding-bottom: 35px;
+    margin-top: 130px;
+}
+
+.Feedback input {
+    border-bottom: 2px solid #000000;
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: you can use single border rule for this

File: css-practice/task 1/style.css [hunk 13]
```diff
@@ -0,0 +1,223 @@
...
+    font-weight: 400;
+    line-height: 24px;
+}
+
+.Feedback button {
+    width: 40px;
+    height: 21px;
+    border: none;
+    background-color: #f0f0f0;
+    background: url(img/send_icon.png);
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: same as above

File: css-practice/task 1/style.css [hunk 14]
```diff
@@ -0,0 +1,223 @@
...
+
+header {
+    margin-top: 75px;
+    margin-bottom: 77px;
+    margin-left: 70px;
+    margin-right: 70px;
+}
+
+footer {
+    width: 100%;
```
Comment (tarasbobak) [inline css-practice/task 1/style.css]: isn't footer a block element with 100% width as default?

File: css-practice/task 1/index.html (hunk 1 above)
Comment (AlexLarusso) [reply css-practice/task 1/index.html]: And don't use UpperCase for classes too.

File: css-practice/task 1/index.html (hunk 3 above)
Comment (AlexLarusso) [reply css-practice/task 1/index.html]: Make it vice versa

**Pull request merged**